### PR TITLE
Fix publishDocumentation & tests cleanup

### PR DESCRIPTION
**Pull request by tchief**
## Changes
 - Fix `publishDocumentation`: on DEV `codegen/workspaces/${version.designSystem.workspaceId}/jobs/documentation` returns different shape
 - Tests cleanup

### CODE REVIEW

File: src/core/data/SDKDataCore.ts [hunk 1]
```diff
@@ -1208,10 +1208,10 @@ export class DataCore {
         scheduledId: string
         exporterId: string
       }
-    } = await this.bridge.postDSMDataToGenericEndpoint(endpoint, payload, false)
+    } = (await this.bridge.postDSMDataToGenericEndpoint(endpoint, payload, false)).result;
```
Comment (tchief) [inline src/core/data/SDKDataCore.ts:1211]: @JiriTrecak Here's the fix itself. Found when run different test:  `test_documentation_publish`

File: src/tests/Tooling/ToolDocSearch.spec.ts [hunk 2]
```diff
@@ -32,5 +32,5 @@ test('test_tooling_doc_search', async t => {
 
     // Search for keyword
     await t.true(searchEngine.search("introduction").length > 0)
-    await t.true(searchEngine.search("ordered").length > 0)
+    await t.true(searchEngine.search("color").length > 0)
```
Comment (tchief) [inline src/tests/Tooling/ToolDocSearch.spec.ts:35]: @JiriTrecak This one was also broken, changed to other search term that working. If it _should be_ `ordered` instead, then I'll mark the test as failing

File: test-resources/figma-tokens/multi-file-sync/supernova.settings.json [hunk 3]
```diff
@@ -2,13 +2,13 @@
     "mode": "multi-file",
     "mapping": [{
             "tokensTheme": "079b44a8b28ed46aff5be1542d750aaa108d08e1",
-            "supernovaBrand": "42d93b8e-ef03-4c0d-bf77-cbbb12a3953a",
+            "supernovaBrand": "Default",
```
Comment (tchief) [inline test-resources/figma-tokens/multi-file-sync/supernova.settings.json:5]: @JiriTrecak Changed specific ids to Default names, hope you don't mind. Now it is easier to test it on new workspaces

**Review by tchief — COMMENTED** [review]
Comments

File: src/tests/Tooling/ToolDocSearch.spec.ts (hunk 2 above)
Comment (JiriTrecak) [reply src/tests/Tooling/ToolDocSearch.spec.ts:35]: All good. This was mostly specific to that one workspace.

File: test-resources/figma-tokens/multi-file-sync/supernova.settings.json (hunk 3 above)
Comment (JiriTrecak) [reply test-resources/figma-tokens/multi-file-sync/supernova.settings.json:5]: I definitely appreciate this change 👍🏻

File: src/core/data/SDKDataCore.ts (hunk 1 above)
Comment (JiriTrecak) [reply src/core/data/SDKDataCore.ts:1211]: Thank you for finding it 🙏🏻

**Comment by JiriTrecak** [comment]
🚢

**Pull request merged**